### DISCUSSION TITLE
Please tell me some singers.

### DISCUSSION THREAD

**Discussion by eito-sa** [NAIST-SE/AttractingContributors#95] [General]
I like singers like Susumu Hirasawa.
If you have any recommendations, please let me know.

This is my site.
https://github.com/eito-sa/AttractingContributors-singers

**Comment by sergiodl-naist** [comment]
Added REIS from DADARAY
Please visit and try to ask a question or recommend me one vtuber on my fork!
https://github.com/sergiodl-naist/AttractingContributors-VTubers

**Comment by shota-in** [comment]
I added.
Please help my task too.
https://github.com/shota-in/AttractingContributors-Songs

**Comment by iikyara** [comment]
I added!
Please help me too!
https://github.com/iikyara/AttractingContributors-Gifts

**Comment by takara-f** [comment]
I added Mroon5 !
Please help me if you can.
https://github.com/F-TKR/AttractingContributors-game
